Veritrans/Config.php

```diff
@@ -1,17 +1,47 @@
 <?php
-
+/**
+ * Veritrans Configuration
+ */
 class Veritrans_Config {
 
+  /**
+   * Your merchant's server key
+   * @static
+   */
   public static $serverKey;
+  /**
+   * Your merchant's client key
+   * @static
+   */
   public static $client_key;
+  /**
+   * This should be 2
+   * @static
+   */
   public static $apiVersion = 2;
+  /**
+   * Make it true to integrate with production
+   * @static
+   */
   public static $isProduction = false;
+  /**
+   * Enable 3D Secure by default
```

Comment:
This key is to enable/disable 3d secure by default. Have better idea about description?